config/configManager.go

```diff
@@ -31,6 +31,7 @@ type GetExecutor func(context.Context) *ef.Executor
 var (
 	configHash string
 	executor   ef.Executor
+	CRLConf    CRLConfig
 )
 
 // Create a executor from configurationFile and setup config file watcher
```

Comment:
this is called on line 111, which requires `opts.enableCrdManager` is false, will this work when it's true?
https://github.com/ratify-project/ratify/blob/dev/cmd/ratify/cmd/serve.go#L111

Reply:
updated.